opt_einsum/helpers.py

```diff
@@ -167,3 +168,95 @@ def flop_count(idx_contraction, inner, num_terms, size_dictionary):
         op_factor += 1
 
     return overall_size * op_factor
+
+
+def rand_equation(n, reg, n_outer, dmin=2, dmax=9, seed=None):
+    """Generate a random contraction and shapes.
+
+    Parameters
+    ----------
+    n : int
+        Number of array arguments.
+    reg : int
+        Average connectivity of graph.
+    n_outer : int
+        Number of outer indices.
+    dmin : int, optional
+        Minimum dimension size.
+    dmax : int, optional
+        Maximum dimension size.
+    seed: int, optional
+        If not None, seed numpy's random generator with this.
+
+    Returns
+    -------
+    eq : str
+        The equation string.
+    shapes : tuple[tuple[int]]
+        The array shapes.
+
+    Examples
+    --------
+    >>> eq, shapes = rand_equation(n=10, reg=4, n_outer=5, seed=42)
```

Comment:
Should be `rand_equation(10, 4, 5, seed=42)` I think?

Reply:
This works fine for me leaving them all as keyword arguments?